index.html

```diff
@@ -1,31 +1,201 @@
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <li class="lingos">html</li>
+                    <li class="lingos">css</li>
+                    <li class="lingos">javascript</li>
+                </ul>
+            </div>
+            <div class="oversight">See-Project</div>
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <li class="lingos">html</li>
+                    <li class="lingos">css</li>
+                    <li class="lingos">javascript</li>
+                </ul>
+            </div>
+            <div class="oversight">See-Project</div>
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <li class="lingos">html</li>
+                    <li class="lingos">css</li>
+                    <li class="lingos">javascript</li>
+                </ul>
+            </div>
+            <div class="oversight">See-Project</div>
+        </div>
+        <div class="container">
+            <div class="container1">
+                <h2 class="cont-header">Professional Art Printing Data</h2>
+                <p class="cont-text"> A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                    has been the industry's standard
+                </p>
+                <ul class="lang">
+                    <li class="lingos">html</li>
+                    <li class="lingos">css</li>
+                    <li class="lingos">javascript</li>
+                </ul>
+            </div>
+            <div class="oversight">See-Project</div>
+        </div>
+    </section>
+    
+    <section class="About-me">
+        <div class="picfix">
+        <div class="abt">
+            <h3 class="bou-me">About me</h3>
+            <p class="abt-text">Hello I’m a software developer!
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+            <button class="resume">Get My Resume</button>
+        </div>
```

Comment:
- [ ] kindly make sure the `Get My Resume` button follow the button styling: **Active**, **Hover** and **Pressed** (see design) as from Figma design.
You can find every styling detail on the below image from your chosen **[Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)**:
![buttons_2](https://user-images.githubusercontent.com/83042742/185633596-7ee998e3-d30d-43c3-9c6a-cfcc6c6be23c.png)